src/apps/concepts/components/NamesTable.tsx

```diff
@@ -45,6 +45,10 @@ const useStyles = makeStyles((theme: Theme) =>
     },
     minimumCellWidth: {
       minWidth: "150px"
+    },
+    menuItemSynonym: {
+      color: "#A9A9A9",
+      fontStyle: "italic"
```

Comment:
I'm not sure about the font style. Also, since we use the Material Design System for this app, we should consult their guidelines about color. In particular, for disabled text, instead of adjusting the color directly, the [Material guidelines](https://material.io/design/color/text-legibility.html#text-backgrounds) suggest using 38% percent opacity.

That means this should probably be:

```suggestion
      opacity: 0.38
```